modules/ximgproc/test/test_letterbox_resize.cpp

```diff
@@ -0,0 +1,33 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+#include "test_precomp.hpp"
+
+namespace opencv_test { namespace {
+
+TEST(ximgproc_letterboxResize, regression)
+{
+    string img_path = string(cvtest::TS::ptr()->get_data_path()) + "fruits.jpg";
```

Comment:
your test image does not exist in the opencv_extra repo (thus the test fails)
it would also need the intermediate folders, like:

`string img_path = string(cvtest::TS::ptr()->get_data_path()) + "cv/npr/test3.png" `

Reply:
I fixed the issue and pushed the change yesterday. When will the PR rebuilt re-run?

Reply:
it already happened, and resp. bots are all green ;)